tests/catalog/test_dynamodb.py

```diff
@@ -579,6 +580,74 @@ def test_passing_provided_profile() -> None:
         assert test_catalog.dynamodb is mock_session().client()
 
 
+@mock_aws
+def test_passing_glue_session_properties() -> None:
+    session_properties: Properties = {
+        "dynamodb.access-key-id": "dynamodb.access-key-id",
+        "dynamodb.secret-access-key": "dynamodb.secret-access-key",
+        "dynamodb.profile-name": "dynamodb.profile-name",
+        "dynamodb.region": "dynamodb.region",
+        "dynamodb.session-token": "dynamodb.session-token",
+        "client.access-key-id": "client.access-key-id",
+        "client.secret-access-key": "client.secret-access-key",
+        "client.region": "client.region",
+        "client.session-token": "client.session-token",
+        "aws_access_key_id": "aws_access_key_id",
```

Comment:
nit: can make these reuseable. i noticed this is used frequently
```
        "aws_access_key_id": "aws_access_key_id",
        "aws_secret_access_key": "aws_secret_access_key",
        "aws_session_token": "aws_session_token",
        "region_name": "region_name",
        "profile_name": "profile_name",
```

Perhaps
```
deprecated_properties = {
        "aws_access_key_id": "aws_access_key_id",
        "aws_secret_access_key": "aws_secret_access_key",
        "aws_session_token": "aws_session_token",
        "region_name": "region_name",
        "profile_name": "profile_name",
}

session_properties = {
        ...,
        **deprecated_properties
} 
```

Reply:
Thanks for the suggestion! I've extracted deprecated properties and unified configurations (`client.*) to `conftest.py`